paths.tex

```diff
@@ -13,48 +15,118 @@
 \numberwithin{thm}{subsection}
 
 \begin{document}
-
-\begin{center} \Large Shortest Path definitions\end{center}
+\title{Temporal Graphs Research at Pomona College\\
+  \textit{Working Definitions and Vocabulary}
+}
+\maketitle
+
+\section{About}
+
+As we continue researching temporal graphs and their related properties there is
+an increased need to establish fundamental definitions. Although many papers
+have established definitions of the terms and concepts contained within this
+paper, there are subtle details of each of these definitions that vary from
+paper to paper making it difficult to efficiently communicate exact ideas.
+
+This document shall serve as an ongoing record of the definitions we will use
+in our research. The definitions contained within should serve as the defaults
+in conversations; if you intend to use an alternate definition to a concept
+defined in this document, you should state and/or cite the alternate definition.
+(In the future, we will look to incorporate alternate definitions in this
+document as well.) It should be noted that these definitions may change as we
+continue our research.
 
 \section{Preliminary Definitions}
 
-Last week we talked about several potential definitions of `temporal shortest
-path' that have different semantic value. We will consider these definitions
-and address their usefulness by examining algorithms and attempting to prove
-different facts.
+\begin{defn}
+  A \textbf{temporal graph} or \textbf{temporal network}\footnote{We will treat
+  the terms \textit{graph} and \textit{network} synonymously throughout this
+  paper.} is defined as a tuple $G \in V \times E$ where $V$ is the set of
+  vertices, and $E \in V^2 \times T^2$ is the set of edges such that
+  $T = [0, \infty)$:
+
+  $$G = (V, E)$$
+
+  Each edge $e \in E$ is a 5-tuple $(v_i, v_j, t_0, t_f, w)$ such that
+  $v_i, v_j \in V$, $t_0, t_f \in T$, and $t_0 \leq t_f$. An edge is said to
+  connect $v_i$ to $v_j$ (with the possibility of $v_i = v_j$) between the time
+  $[t_0, t_f]$ with weight $w$. That is, all edges are to be treated as directed
+  edges with some weight and time-window (possibly instantaneous) in which they
+  are \textit{active}.
+\end{defn}
+
+With this generalized definition of a temporal network, we will begin to explain
+some common, specialized descriptors of these networks:
+
+\begin{defn}
+  We will describe a temporal graph as \textbf{undirected} $\forall e \in E$
+  if there exists $e' \in E$ such that $e = (v_i, v_j, t_0, t_f,w)$ and $e' =
+  (v_j, v_i, t_0, t_f, w)$.\footnote{As we consider the real-life applications
```

Comment:
Yes -- we probably want to discuss this further and consider the implications.

For theory I think it makes most sense as an unordered pair, but in application it is convenient to define an edge in both directions and adapt the logic and other theorems to avoid the caveats that you pointed out. I'm not sure which is ultimately better. I like the idea of having the generic definition and this allows for that, but also understand it create complications elsewhere.